foundry.toml

```diff
@@ -14,6 +14,7 @@ remappings = [
   '@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts',
   '@base-contracts/=lib/base-contracts/',
   '@eth-optimism-bedrock/=lib/optimism/packages/contracts-bedrock/',
+  '@op/=lib/optimism/packages/contracts-bedrock/',
```

Comment:
nit: an `@eth-optimism-bedrock` monorepo mapping already exists. Let's use that.

Reply:
I added it because `eth-optimism-bedrock` is so unnecessarily long lol